tests/install/openqa_webui.pm

```diff
@@ -12,7 +12,9 @@ sub add_repo {
     );
     my $repo = 'openSUSE_' . $repo_suffix{get_required_var('ARCH')};
     my $project = get_var('OPENQA_OBS_PROJECT', 'devel:openQA');
-    my $repo_url = get_var('OPENQA_REPO_URL', "obs://$project/$repo");
+
+    # Using PEDANTIC here to skip the redirector service
```

Comment:
Now this comment does not apply anymore. Please update it to also reflect that this only done as an exceptional case as we work with a transient repository content.